sparkles/tests/test_checks.py

```diff
@@ -308,6 +308,33 @@ def test_guide_count_or():
     assert msg['category'] == 'caution'
     assert 'OR with more than 1 stars brighter than 5.5.' in msg['text']
 
+    # Set a scenario with guide_count in the 3.5 to 4.0 range and confirm a
+    # critical warning.
+    stars = StarsTable.empty()
+    stars.add_fake_constellation(n_stars=5, mag=[7.0, 7.0, 7.0, 10.2, 10.3])
+    aca = get_aca_catalog(**mod_std_info(n_fid=3, n_guide=5, obsid=1),
+                          stars=stars, dark=DARK40,
+                          raise_exc=True)
+    aca = ACAReviewTable(aca)
+    aca.check_guide_count()
+    assert aca.messages == [
+        {'text': 'OR count of guide stars 3.65 < 4.0', 'category': 'critical'},
+        {'text': 'OR with 4 guides but 5 were requested', 'category': 'caution'}]
+
+    # Set a scenario with guide_count in the 3.5 to 4.0 range and *but* with
+    # a creep away (maneuver angle <= 5), and confirm that is just a warning
+    # (not critical).
+    stars = StarsTable.empty()
+    stars.add_fake_constellation(n_stars=5, mag=[7.0, 7.0, 7.0, 10.2, 10.3])
```

Comment:
You can re-use the existing `stars`.

Reply:
Actually, looking at this again, all of these individual tests within this function are independent. So they should each be a separate test. That way if one fails then you still test all the rest.

(My understanding of best practices for writing tests continues to improve and I now have a better appreciation for making unit tests atomic where possible).